src/generator/generator_visitor.ts

```diff
@@ -370,9 +371,17 @@ class GeneratorVisitor extends Visitor {
 
         return index;
     }
+
+    private getAdress() {
```

Comment:
Rename this method to `getArrayPointer()` because it's not the address that is returned